src/common/mem.c

```diff
@@ -34,7 +34,7 @@ struct umem_tx_stage_item {
 
 #ifdef DAOS_PMEM_BUILD
 
-static int use_bmem = 1; /* Use BMEM by default */
+static int use_bmem; /* Don't use BMEM by default */
```

Comment:
We currently run vos_tests with bmem in CI (see utils/run_test.sh), this change will disable bmem even when DAOS_MD_ON_SSD=1 is exported (see umempobj_settings_init()).